consul/server_manager/server_manager.go

```diff
@@ -47,11 +47,19 @@ const (
 	newRebalanceConnsPerSecPerServer = 64
 )
 
+// ConsulClusterInfo is an interface wrapper around serf and prevents a
+// cyclic import dependency
 type ConsulClusterInfo interface {
 	NumNodes() int
 }
 
-// serverCfg is the thread-safe configuration struct used to maintain the
+// ConnPoolTester is an interface wrapping client.ConnPool to prevent a
+// cyclic import dependency
+type ConnPoolPinger interface {
```

Comment:
The interface has been renamed, the struct member was not however in order to hint that the pinger is bringing up a connection that remains resident in the `ConnPool`.